### PR TITLE
Easy tests reconst

### PR DESCRIPTION
**Pull request by EmmaRenauld**
# Quick description

Took a few minutes today to prepare some unit test in the reconst module. 

Coverage of unit test changes (only; no script tests) are:
`reconst.fodf`: from 0% to 36%
`reconst.frf`: from 0% to 49%
 

Unchanged reconst files:
`reconst.aodf` = 0% (@CHrlS98 if you want to contribute)!
`reconst.bingham` = 88% (Thanks Charles)
`reconst.divide, reconst.mti, reconst.sh` = 0% (@karanphil if you want to contribute!?)
`reconst.utils`: 36%

## Type of change

Check the relevant options.

- [x] Unit tests

# Checklist

- [x] My code follows the style guidelines of this project (run [autopep8](https://pypi.org/project/autopep8/))
- [ ] I added relevant citations to scripts, modules and functions docstrings and descriptions
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I moved all functions from the script file (except the argparser and main) to scilpy modules
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: scilpy/reconst/fodf.py [hunk 1]
```diff
@@ -57,26 +63,17 @@ def get_ventricles_max_fodf(data, fa, md, zoom, sh_basis, small_dims,
...
-            step = 5
-
     # 1000 works well at 2x2x2 = 8 mm3
     # Hence, we multiply by the volume of a voxel
     vol = (zoom[0] * zoom[1] * zoom[2])
     if vol != 0:
         max_number_of_voxels = 1000 * 8 // vol
     else:
         max_number_of_voxels = 1000
+    logging.debug("We will search ventricle voxels, up to a maximum of {} "
```
Comment (karanphil) [inline scilpy/reconst/fodf.py:75]: "Searching for ventricle voxels" perhaps? Not a fan of "we will"

File: scilpy/reconst/frf.py [hunk 2]
```diff
@@ -19,10 +19,13 @@
...
-    Response Function from a DWI volume.
-    A DTI fit is made, and voxels containing a single fiber population are
-    found using a threshold on the FA.
+    """
+    Computes a single-shell (under b=1500), single-tissue single Fiber
+    Response Function from a DWI volume. A DTI fit is made, and voxels
+    containing a single fiber population are found using either a threshold on
+    the FA, inside a white matter mask.
+
+    Requires
```
Comment (karanphil) [inline scilpy/reconst/frf.py:28]: Requires what?

**Review by karanphil — APPROVED** [review]
Two small comments and it's GTG!

**Review by arnaudbore — APPROVED** [review]
GTG

**Pull request merged**